examples/model_selection/plot_train_error_vs_test_error_and_validation_curve.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
It seems that we don't need this file because it corresponds to the file above. I think that we should not rename any file here.

Reply:
Noted, will make the changes, how about header for this example would it be good like this:
 `Train error vs Test error and Validation Curve`